yapapi/contrib/strategy/provider_filter.py

```diff
@@ -60,7 +60,7 @@ def denying_event_consumer(event: events.Event):
     """
 
     def __init__(self, base_strategy: MarketStrategy, is_allowed: IsAllowedType):
```

Comment:
I think we should have `base_strategy: MarketStrategyType` here. There's no reason not to wrap a wrapper, I hope?

Reply:
yes, I _just_ realized it too :)